### PR TITLE
feat : added a  navigation component

### PR DESCRIPTION
**Pull request by GeetaPogul**

### CODE REVIEW

File: CSS/slidebar.css [hunk 1]
```diff
@@ -24,15 +25,16 @@ a{
 .list-links{
     
     align-items: center;
-    padding: 1.2rem 1.1rem;
+    padding: 1.2rem 1.2rem;
```
Comment (someshtiwarii) [inline CSS/slidebar.css]: can use  (padding: 1.2 rem;) only

File: Navigation/nav.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+.nav-box{
+    padding: .3rem;
+    height: 10rem;
+    width: 100%;
+}
+
+.right-side-links{
+    margin-left:14rem;
+
```
Comment (someshtiwarii) [inline Navigation/nav.css:12]: can use prettier for clean formatting of code

**Review by someshtiwarii — APPROVED** [review]
little bit of formatting can be done , rest looks good !

File: Navigation/nav.html [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+                <li class="list-links"> <a href="../Typography/typography.html" >  Typography</li> </a>
+                <li class="list-links"> <a href="../Navigation/nav.html" >  Navigation</li> </a>
+            </ul>
+          </div>
+      
+      
+        <div class="doc-content">
+         <div class="content-title" > Navigation</div>
+         
+         <br> <br>
```
Comment (sakshamAK) [inline Navigation/nav.html]: This is not a good practice (and also br is an empty tag i.e. it should be like &lt;br /&gt;), instead, you can use margin here.

File: Navigation/nav.html [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+               
+               
+                           <li  class="nav-li-links">   <span >  <button class="theme-button" id="theme-btn" > 
+                              <i class='fas fa-moon'> </i>   </button>
+                           </li>
+                       </ul>
+                 
+               </div>
+              </nav> 
+           </div> <br>
```
Comment (sakshamAK) [inline Navigation/nav.html]: again try not to add br tag for margins.
https://webmasters.stackexchange.com/questions/79199/why-isnt-the-html-break-tag-acceptable-anymore-for-top-and-bottom-margins
check this out.

File: Navigation/nav.html [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+         <br> <br>
+         <div class="content-subtitle" > 
+    
+            Nav bar displays the list of choices for the user avaible of the web page.
+            </div>
+ <hr>
+             
+             
+             
+         <!-- <div class="circle-avatar"> -->
```
Comment (sakshamAK) [inline Navigation/nav.html]: if this tag is not needed you can remove that.

File: Navigation/nav.html [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+      
+        <div class="doc-content">
+         <div class="content-title" > Navigation</div>
+         
+         <br> <br>
+         <div class="content-subtitle" > 
+    
+            Nav bar displays the list of choices for the user avaible of the web page.
+            </div>
+ <hr>
```
Comment (sakshamAK) [inline Navigation/nav.html]: make it &lt;hr /&gt;

**Review by sakshamAK — APPROVED** [review]

**Pull request merged**